### PR TITLE
Add windows docs

### PR DESCRIPTION
**Pull request by hoosteeno**
Fixes https://github.com/mozilla/shield-studies-addon-template/issues/29

### CODE REVIEW

File: WINDOWS_SETUP.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+## Requirements
+
+* Windows Version 10.0.14986+ (find your version by typing `ver` at Windows command line)
+
+## Installing WSL
+
+1. [Follow Microsoft's official steps for installing WSL](https://answers.microsoft.com/en-us/insider/wiki/insider_wintp-insider_install/how-to-enable-the-windows-subsystem-for-linux/16e8f2e8-4a6a-4325-a89a-fd28c7841775?auth=1). These instructions are clear and detailed. _If you prefer, here is a TL;DR:_
+    1. Enable developer mode in Windows 10 in `Start > Settings > Update & security > For developers`.
+    2. Enable the optional Windows feature, "Windows Subystem for Linux" using `optionalfeatures.exe`.
```
Comment (pdehaan) [inline WINDOWS_SETUP.md]: typo: "Subsystem"

Comment (hoosteeno) [reply WINDOWS_SETUP.md]: Fixed in latest push.

File: WINDOWS_SETUP.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+```
+3. Get the latest npm: `npm install npm@latest -g`
+
+4. Install git and zip (and any other linux command-line tools you like):
+```
+sudo apt install git
+sudo apt install zip
+```
+
+5. Now you can follow the standard instructions in the README at [https://github.com/mozilla/shield-studies-addon-template](https://github.com/mozilla/shield-studies-addon-template) for cloning the repo, installing npm modules and running npm scripts.
```
Comment (biancadanforth) [inline WINDOWS_SETUP.md]: Nit: Could you use a relative link here?
`... instructions in the [README](/README.md)`

Comment (biancadanforth) [reply WINDOWS_SETUP.md]: Actually I changed my mind, sorry. This is fine. I don't know that there is a relative URL for the repo home. The link above wouldn't have the desired effect.

**Pull request merged**